### DISCUSSION TITLE
Split pane to vertical or horizontal on the current window like iTerm2?

### DISCUSSION THREAD

**Discussion by lamhuynhit** [isontheline/pro.webssh.net#1089] [Q&A]
Could you please split pane to vertical or horizontal on the current window like iTerm2? 

![Screenshot 2024-04-01 at 10 33 17](https://github.com/isontheline/pro.webssh.net/assets/7575497/33a6f686-fde1-4cd8-803d-9fb70f0597ff)

**Comment by isontheline** [comment]
Hello @lamhuynhit 👋

Thanks for your feedback 🙏

It's not a trivial task but I will try to implement it in a future release of WebSSH on macOS.

I've just create an issue with your request : https://github.com/isontheline/pro.webssh.net/issues/1090

Have a nice day ☀️
